yarn.lock

```diff
@@ -2884,6 +2903,134 @@
+  version "9.0.1"
+  resolved "https://registry.yarnpkg.com/@styled-icons/boxicons-solid/-/boxicons-solid-9.0.1.tgz#044907ce675db30250441805d1a8fe5e9612106d"
+  integrity sha512-GzUq4pLLqEjQWHY7ayGjikLUAfntVQpYIoLwvEa0wYh+8oJLk0+VOlbZRCAXg5Yye/pMMySTgr4uO8i1nXOdlg==
+  dependencies:
+    "@styled-icons/styled-icon" "^9.0.1"
+    tslib "^1.9.3"
+
+"@styled-icons/crypto@^9.0.1":
+  version "9.0.1"
+  resolved "https://registry.yarnpkg.com/@styled-icons/crypto/-/crypto-9.0.1.tgz#fd03a711c7f3416413b268fc6225c3e7917b44ea"
+  integrity sha512-gfOzMu4zqXZNSlWgcu5+psdUwuwBL8ajMpGwqZRjC4LAEuYMyJV8NWsd6XY7UEVXSz31xH+81Az75gKElN5hMg==
+  dependencies:
+    "@styled-icons/styled-icon" "^9.0.1"
+    tslib "^1.9.3"
+
+"@styled-icons/evil@^9.0.1":
+  version "9.0.1"
+  resolved "https://registry.yarnpkg.com/@styled-icons/evil/-/evil-9.0.1.tgz#1a2c3060dae4434928ad0684a68ac114da49474b"
+  integrity sha512-r/C6XvO4iOXBvvWOC31GDozvj6WmyoYVfzZm5g9bTWhkbdmAJf+w4+ZHb8QnG7tpZ1XvEwbd8VrFIU70e9DLEg==
+  dependencies:
+    "@styled-icons/styled-icon" "^9.0.1"
+    tslib "^1.9.3"
+
+"@styled-icons/fa-brands@^9.1.0":
+  version "9.1.0"
+  resolved "https://registry.yarnpkg.com/@styled-icons/fa-brands/-/fa-brands-9.1.0.tgz#3604e56a57894bf445141ef965e1e9276eaec387"
+  integrity sha512-wBwiYSv7lcGpYG3rWmlRZbP1sbikc7nMgxKKk0nHJFdYM5jwv4npsKav2TXtN2BJlMvuYeU+eeud/EaNwFqFhA==
+  dependencies:
+    "@styled-icons/styled-icon" "^9.0.1"
+    tslib "^1.9.3"
+
+"@styled-icons/fa-regular@^9.1.0":
+  version "9.1.0"
+  resolved "https://registry.yarnpkg.com/@styled-icons/fa-regular/-/fa-regular-9.1.0.tgz#3f01804b37d89c92f36d18e5fbdc89f99741dfad"
+  integrity sha512-k0RqQp14XbvFqRFNm5DiqEbP6kRWkoebmeIwxsjEbpr4p+Yj+leC8aVrF8LBtDPgjrmLIWWuIdzZDKhRXnRujA==
+  dependencies:
+    "@styled-icons/styled-icon" "^9.0.1"
+    tslib "^1.9.3"
+
+"@styled-icons/fa-solid@^9.1.0":
+  version "9.1.0"
+  resolved "https://registry.yarnpkg.com/@styled-icons/fa-solid/-/fa-solid-9.1.0.tgz#0cff781702bf8a0fc742840bc0e62d551d1508ab"
+  integrity sha512-rB0HDzp3F2XFoYMTWlWiYBcSIDOA2Y/NfBFEKo+G0mWH95etNecuSwRrERjdRYq0wWbtUJ8knrZerJY2CguXeQ==
+  dependencies:
+    "@styled-icons/styled-icon" "^9.0.1"
+    tslib "^1.9.3"
+
+"@styled-icons/feather@^9.1.0":
+  version "9.1.0"
+  resolved "https://registry.yarnpkg.com/@styled-icons/feather/-/feather-9.1.0.tgz#ae05eefa9c5157c5c46eccc327da5eeb6dba4d2f"
+  integrity sha512-UCvpRgsIvpuUXVYvZhlxaNXjeF4C40FWL3GaojZXcHpgAs3KHVzp3u0pToYqZNO4mz+0oEXe1m13CPbkYegs5w==
+  dependencies:
+    "@styled-icons/styled-icon" "^9.0.1"
+    tslib "^1.9.3"
+
+"@styled-icons/icomoon@^9.0.1":
+  version "9.0.1"
+  resolved "https://registry.yarnpkg.com/@styled-icons/icomoon/-/icomoon-9.0.1.tgz#43b97acf25f89aaf78918b968142d867bf2efaba"
+  integrity sha512-PHXCeq95ODVEz4TpEZKlKwOcWKRRqnFmqi6iqcD++EMaNfUQvWSvs0aP+724VqfkPrNmyTFvjn16toZlTAHrsg==
+  dependencies:
+    "@styled-icons/styled-icon" "^9.0.1"
+    tslib "^1.9.3"
+
+"@styled-icons/material@^9.0.1":
```

Comment:
should we only install the ones we need? i.e. only `@styled-icons/material` so far?
isnt really necessary

Reply:
Yes, forgot to remove the rest of the library as i was experimenting with other icons